tests/functional/tests/eviction/test_eviction.py

```diff
@@ -20,17 +20,43 @@
 logger = logging.getLogger(__name__)
 
 
+@pytest.mark.parametrize("cls", CacheLineSize)
+@pytest.mark.parametrize("mode", [CacheMode.WT])
+def test_two_cores(pyocf_ctx, mode: CacheMode, cls: CacheLineSize):
+    """Test if eviction does not occur when IO greater than cache size is submitted."""
```

Comment:
Based on the test below, shouldn't this one be called something like `test_write_size_greater_than_cache_with_two_cores`? And also update the description?

Reply:
right, fixe